libc/test/src/string/StrchrTest.h

```diff
@@ -114,25 +117,25 @@ template <auto Func> struct StrrchrTest : public LIBC_NAMESPACE::testing::Test {
     const char *src = "abcde";
 
     // Should return null terminator.
-    ASSERT_STREQ(Func(src, '\0'), "");
+    ASSERT_TRUE(Func(src, '\0') != nullptr);
```

Comment:
I think this hunk specifically no longer aligns with the commend above it?

---

We do define a (currently unused) `ASSERT_STRNE`.

Reply:
maybe `Func(src, '\0') != nullptr && *Func(src, '\0') == '\0'`?

Reply:
Ah, good catch. I mirrored the `// Should return null terminator. change from above.` Happy to do `Func(src, '\0') != nullptr && *Func(src, '\0') == '\0'` if you all would prefer; just want to keep the two in sync